notebooks/Hugging_Face_Finetuning.ipynb

```diff
@@ -0,0 +1,920 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    TOKENIZER = "bert-base-uncased"
</pre><p>Should we the distilbert-base-uncased tokenizer here?</p><p> </p><p>XREF: <a href="https://huggingface.co/distilbert-base-uncased" target="_blank">https://huggingface.co/distilbert-base-uncased</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/skorch-dev/skorch/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Changed.